docs/contributing/01-start-here/05-development/_category_.yml

```diff
@@ -0,0 +1,3 @@
+label: Development
+collapsible: true
+collapsed: true
```

Comment:
I wonder if we can get away with just having a snippet in the development page rather than turning it into a collapsable section.